src/main/java/it/pagopa/pn/paperchannel/utils/FeedbackStatus.java

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.pn.paperchannel.utils;
+
+import lombok.NonNull;
+
+/**
+ * This class groups and centralizes attributes related to feedback.
+ *
+ * @param oldFeedbackDeliveryFailureCause There may not have been a delivery failure cause, so it can be null.
+ */
+public record FeedbackStatus(@NonNull String oldFeedbackStatusCode, @NonNull String newFeedbackStatusCode,
+                             @NonNull String oldFeedbackStatusDateTime, @NonNull String newFeedbackStatusDateTime,
+                             String oldFeedbackDeliveryFailureCause, String newFeedbackDeliveryFailureCause) {
```

Comment:
@Vallasc togliamo le annotazioni "NonNull" perché potrebbero comunque verificarsi casi in cui i valori sono nulli, però ci proteggiamo con il controllo nel commento sopra